cypress/integration/homepage_spec.js

```diff
@@ -0,0 +1,33 @@
+describe('Main Page Render', () => {
+    beforeEach(() => {
```

Comment:
Do we want to do an intercept and stub out 1 country to check for in the dropdown?